tbconnect/tests/test_views.py

```diff
@@ -102,6 +102,33 @@ def test_location_validation(self):
             {"non_field_errors": ["location and city_location are both None"]},
         )
 
+    def test_skip_location_validation(self):
+        """
+        Should create a new TBCheck object in the database
+        """
+        user = get_user_model().objects.create_user("test")
+        user.user_permissions.add(Permission.objects.get(codename="add_tbcheck"))
+        self.client.force_authenticate(user)
+        response = self.client.post(
+            self.url,
+            {
+                "msisdn": "27856454612",
+                "source": "USSD",
+                "province": "ZA-WC",
+                "city": "Cape Town",
```

Comment:
is city optional?

Reply:
No it's not optional, it is mandatory.

Reply:
![image](https://user-images.githubusercontent.com/95408319/178974137-fc2aad85-dcec-4e6b-bae0-f58d50773fdc.png)

Reply:
when we skip location, do we also skip city question?

Reply:
then the tbcheck POST will fail